### PR TITLE
Add database setup

### PR DESCRIPTION
**Pull request by goh0222**
database

### CODE REVIEW

File: db/migrate/20220831102058_add_index_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddIndexToUsers < ActiveRecord::Migration[7.0]
```
Comment (Rainesisu) [inline db/migrate/20220831102058_add_index_to_users.rb:1]: note to self: User = Parent model in db table

File: db/schema.rb [hunk 2]
```diff
@@ -33,6 +33,37 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "child_activities", force: :cascade do |t|
+    t.bigint "child_id", null: false
```
Comment (Rainesisu) [inline db/schema.rb:37]: @goh0222 question: bigint -> is Primary key for active record, is it the same as foreign key for our db table?
image

File: db/seeds.rb [hunk 3]
```diff
@@ -5,9 +5,22 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Child.delete_all
+User.delete_all
 Activity.delete_all
 Category.delete_all
```
Comment (Rainesisu) [inline db/seeds.rb:12]: familiar names in users and children table =)

File: db/schema.rb (hunk 2 above)
Comment (goh0222) [reply db/schema.rb:37]: this data type for primary key is auto assigned when we run `rails db:migrate`, so just use it as it is.

Comment (goh0222) [reply db/schema.rb:37]: To answer your question, I think it is.

**Review by Yaamanni — APPROVED** [review]

**Pull request merged**